src/test/packages/01-uds-bundle/uds-bundle.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 zarf-packages:
   - name: init
     repository: localhost:888/init
-    ref: v0.29.1
+    ref: v0.30.1
```

Comment:
not to beat a dead horse, but this is why bundle YAML templating needs to be a feature, as this is now 3+ places that need to be updated all at the same time. There may be a way to configure renovate to do this.